spec.html

```diff
@@ -32966,7 +32972,7 @@ <h1>
           GetNamedTimeZoneOffsetNanoseconds (
             _timeZoneIdentifier_: a String,
             _epochNanoseconds_: a BigInt,
-          ): an integer
+          ): an integer in the exclusive interval from -8.64 × 10<sup>13</sup> to 8.64 × 10<sup>13</sup>
```

Comment:
I would prefer defining this type up in Time Values and Time Range alongside "local-offset time value" as something like "timezone offset".

Reply:
We can reuse that definition in Temporal.